api/api_test.go

```diff
@@ -80,10 +115,32 @@ func newTestServer(t *testing.T) *testServer {
 		t.Fatalf("err: %s", err)
 	}
 
+	return &testServer{
+		pid:        cmd.Process.Pid,
+		dataDir:    dataDir,
+		configFile: configFile.Name(),
+	}
+}
+
+func makeClient(t *testing.T) (*Client, *testServer) {
```

Comment:
I kept it that way since that's how it was defined before. I'll change it to a DefaultConfig method.